### DISCUSSION TITLE
Charts colours

### DISCUSSION THREAD

**Discussion by DuvelCorp** [CyferShepard/Jellystat#197] [General]
Hi

I'd like to know if there's a way to customize the colours used in the Statistics charts and if not if something could be done on your side to improve them.

Basically here's my problem:

![image](https://github.com/CyferShepard/Jellystat/assets/33682764/1a582366-4206-4925-8b80-edee88e1da9d)

-> We can barely see any difference with the actual colors chosen.

**Comment by CyferShepard** [comment]
Hey @DuvelCorp , I made some changes to the default color scheme on `unstable`, changing colors will be coming in the next update
